File: chapter5_highest_value.py
votes = []

for line in open('candidates.csv'):
  line = line.split(',')
  candidate_votes = int( line[3] )
  votes.append( candidate_votes )

votes.sort() ## the lowest value is now in index 0

print( "Highest votes", votes[-1] )
print( "Second-highest votes", votes[-2] )

File: chapter5_highest_value.r
votes <- vector()
data <- read.csv("candidates.csv", header = F)

for( i in 1:nrow(data) ){
  candidate_vote <- as.numeric( data[i,3] )
  votes <- c( votes, candidate_vote )
}

votes <- sort( votes )
item_count <- length( votes )

print( paste( "Highest votes", votes[ item_count -1 ] ) )
print( paste( "Second-highest votes", votes[item_count -2 ] ) )
